dashboard/test/mailers/email_delivery_observer_test.rb

```diff
@@ -10,23 +10,15 @@ class EmailDeliveryObserverTest < ActiveSupport::TestCase
       {
         metric_name: :EmailSent,
         dimensions: [
-          {name: "Environment", value: CDO.rack_env}
+          {name: "Environment", value: CDO.rack_env},
+          {name: "Action", value: 'new_teacher_email'},
+          {name: "Class", value: 'TeacherMailer'},
         ],
         value: 1
       }
     ]
     Cdo::Metrics.expects(:push).with('ActionMailer', expected_metric)
 
-    # Ignores other ActionMailer metrics.
-    #
-    # We use EmailDeliveryInterceptor and EmailDeliveryObserver to intercept the email delivery
-    # process. When an email is delivered, both the interceptor and observer are triggered,
-    # pushing their metrics to CloudWatch.
-    # To isolate this test to just the observer, we have to ignore any metrics pushed by
```

Comment:
I don't think this code was actually doing anything. We don't need to "isolate" the observers or interceptors using stubs because we only registering either the observer or the interceptor in the `setup` of the unit test.

Let me know if you think I'm mistaken though.